schema.sql

```diff
@@ -55,9 +55,43 @@ CREATE TABLE specializations (
 );
 
 -- Create the visits TABLE
-CREATE TABLE visits (
-    animal_id INTEGER REFERENCES animals(id),
-    vet_id INTEGER REFERENCES vets(id),
-    visit_date DATE,
-    PRIMARY KEY (animal_id, vet_id, visit_date)
+CREATE TABLE visits(
+animal_id INT,
+vet_id INT,
+date_of_visits DATE,
+CONSTRAINT fk_animals
+FOREIGN KEY(animal_id)
+REFERENCES animals(id),
+CONSTRAINT fk_vets
+FOREIGN KEY(vet_id)
+REFERENCES vets(id)
 );
+
+-- Performance Database
+-- Add an email column to your owners table
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+-- Drop previous visits table and recreated
+DROP TABLE visits;
+
+CREATE TABLE visits(
+  id INT GENERATED ALWAYS AS IDENTITY,
+  animal_id INT REFERENCES animals(id),
+  vet_id INT REFERENCES vets(id),
+  date_of_visits DATE,
+  PRIMARY KEY(id)
+);
+
+CREATE INDEX vet_id_index ON visits (vet_id DESC);
+CREATE INDEX animal_id_index ON visits (animal_id);
```

Comment:
- [ ] 👍 Great work on the project! While your vet_id_index and owners_index screenshots look solid, there's a minor issue with the screenshot for animal_id_index. It appears that both screenshots of animal_id were taken after adding the index. To better represent the state and provide a complete comparison, kindly consider taking a screenshot of the animal_id query execution plan before adding the index. You can achieve this by removing the indexes temporarily, running the query, capturing the `explain analyze` output, and then adding the indexes back.

For reference, you can check out an example [here](https://github.com/errea/vet_clinic_database/pull/1#pullrequestreview-763088027). Keep up the excellent work! 👍🔍📸